src/core/layout/text-utils.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @copyright BBC 2021
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+
+export const updateStyleOnFontLoad = textElem => {
+    // ie11?
```

Comment:
I'm not sure if we need to polyfill this- I can't run local Genie in IE11.

Reply:
Just ran this and it crashes
```
Unable to get property 'check' of undefined or null reference
TypeError: Unable to get property 'check' of undefined or null reference
   at updateStyleOnFontLoad (eval code:10:3)
   at text (eval code:41:5)
   at Anonymous function (eval code:24:5)
   at Anonymous function (eval code:29:7)
   at map (eval code:16:5)
   at createTitles (eval code:46:3)
   at createTitle (eval code:19:3)
   at create (eval code:97:7)
   at create (eval code:4:410431)
   at bootScene (eval code:4:409614)
```

I'm wondering if we should just stub `document.fonts` for IE11 so it doesn't crash?

Reply:
Could go one better and switch to a `setTimeout` solution if `!document.fonts`.

Reply:
I really don't want to add browser specific code for stuff like this if we can avoid it.

Tried updating to latest core-js and they haven't added anything for it.

maybe just do a:

```
if (!document.fonts?.check(textElem.style._font)) {
        document.fonts?.ready.then(() => textElem.style.update());
    }
```

Reply:
Agreed. 5fecd82 has this plus a mini refactor following your two other points. TY!